Android/NumberProgressBar/nuget/Xamarin.NumberProgressBar.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.NumberProgressBar</id>
+    <title>NumberProgressBar for Xamarin.MonoAndroid23</title>
+    <version>1.2.0.1</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>A slim progress indicator with a numeric percentage alongside the progress bar.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=875036</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=875040</licenseUrl>
+  </metadata>
+  <files>
+    <file src="output/NumberProgressBar.dll" target="lib/Xamarin.MonoAndroid23" />
```

Comment:
this should be MonoAndroid not Xamarin.MonoAndroid23

Reply:
Issues fixed.